iwyu_output.cc

```diff
@@ -1778,6 +1778,7 @@ enum class LineSortOrdinal {
   PrecompiledHeader,
   AssociatedHeader,
   AssociatedInlineDefinitions,
+  QuotedHeader,
```

Comment:
And this would then become `QuotedInclude`

Reply:
Done.